packages/app/src/debug/empty/DebugEmptyStates.cy.tsx

```diff
@@ -58,6 +59,12 @@ function mountWithGql (component: JSX.Element, gqlOptions?: { debugSlideshowComp
 }
 
 describe('Debug page empty states', () => {
+  context('empty view', () => {
+    it('renders with slot', () => {
+      mountWithGql(<DebugEmptyView title="My Title"><template v-slot:cta="slotProps"></template></DebugEmptyView>)
```

Comment:
This looks like it needs an assertion? Or do we definitely need the test?

Reply:
Thanks for catching that.  I forgot to go back and finish that test.  It is updated now.